### DISCUSSION TITLE
change delay time on runout sensor

### DISCUSSION THREAD

**Discussion by jxer** [mriscoc/Ender3V2S1#747] [General]
it appears there is a 35-1 minute delay for the filament runout sensor to stop printing, due to a custom setup, this is actually too long for me, is there a setting somewhere that changes how fast this fires?

**Comment by TheWebMachine** [comment]
It's not time based, it's length based. Change the filament Runout Distance in Filament Settings menu. If you set to 0mm, it will stop immediately after the filament has cleared the sensor.

**Reply by jxer** [reply]
really appreciate the info.
